From 7219940b5a6e24dcaeb1cd96049aba25ffc81da4 Mon Sep 17 00:00:00 2001
From: Vladimir Oltean <olteanv@gmail.com>
Date: Wed, 3 Sep 2025 21:15:00 +0300
Subject: [PATCH 2/4] pci: layerscape: include missing <linux/delay.h>
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

Fixes the following build error:

drivers/pci/pcie_layerscape_rc.c: In function ‘ls_pcie_wait_for_link’:
drivers/pci/pcie_layerscape_rc.c:43:17: error: implicit declaration of function ‘udelay’ [-Wimplicit-function-declaration]
   43 |                 udelay(LINK_WAIT_TIMEOUT);
      |                 ^~~~~~

To be squashed into commit "pci: ls_pcie: Wait 100ms for Link Up in
ls_pcie_probe".

Signed-off-by: Vladimir Oltean <olteanv@gmail.com>
---
 drivers/pci/pcie_layerscape_rc.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/drivers/pci/pcie_layerscape_rc.c b/drivers/pci/pcie_layerscape_rc.c
index 33bd14ad81a5..c039d9204ce6 100644
--- a/drivers/pci/pcie_layerscape_rc.c
+++ b/drivers/pci/pcie_layerscape_rc.c
@@ -13,6 +13,7 @@
 #include <malloc.h>
 #include <dm.h>
 #include <dm/devres.h>
+#include <linux/delay.h>
 #if defined(CONFIG_FSL_LSCH2) || defined(CONFIG_FSL_LSCH3) || \
 	defined(CONFIG_ARM)
 #include <asm/arch/clock.h>
-- 
2.43.0

